src/scss/_typography.scss

```diff
@@ -24,45 +14,110 @@ html {
 }
 
 h1 {
-  font-size: var(--font-size-2xl);
-  color: white;
-
-  @include breakpoint-up("large") {
-    font-size: var(--font-size-4xl);
-  }
+  font-size: var(--font-size-h1);
+  font-size: clamp(4rem, 1rem + 5vw, var(--font-size-h1));
 }
 
 h2 {
-  font-size: var(--font-size-2xl);
+  font-size: var(--font-size-h2);
+  font-size: clamp(3.8rem, 1rem + 5vw, var(--font-size-h2));
 }
 
 h3 {
-  font-size: var(--font-size-xl);
+  font-size: var(--font-size-h3);
 }
 
 h4 {
-  font-size: var(--font-size-m);
+  font-size: var(--font-size-h4);
 }
 
-h5 {
-  font-size: var(--font-size-xs);
-
-  @include breakpoint-up("large") {
-    font-size: var(--font-size-lg);
-  }
+h3,
+h4 {
+  font-weight: 600;
 }
 
 p,
 li {
-  font-size: var(--font-size-md);
+  font-size: var(--font-size-text);
 }
 
-h1,
-h2,
-h3,
-h4,
-h5,
-h6,
-strong {
-  font-weight: var(--font-weight-bold);
+p {
```

Comment:
There's a lot of stuff in this file that isn't actually typography

Reply:
Good point. I would remove `video` and `.notecard`. Do you think `article` and `summary` declarations need to be moved somewhere else too?